src/components/grid-aware/VideoHeader/VideoHeader.module.css

```diff
@@ -0,0 +1,138 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedBackground {
+  background:
+    linear-gradient(90deg, #000 0%, rgba(0, 0, 0, 0) 100%),
+    center / cover var(--background-image);
```

Comment:
Since this is a bit esoteric, I think it'd be a good idea to write a comment describing how this `--background-image` works. That is, we should describe that it's expected that the user of the component should set an inline style that sets this CSS custom property.

Reply:
Okay